scss/_root.scss

```diff
@@ -1,3 +1,4 @@
+// Do not forget to update getting-started/theming.md!
```

Comment:
Had better update the theming.md in this PR to not forget?

Reply:
ops, I saw the theming.md but I could not find what this comment means 🤔

Reply:
What do you mean? I added this comment because on v4-dev the contents of _root.scss are not automatically pulled from the file like we do on master. So, if _root.scss changes, we need to manually update that page.

Reply:
https://getbootstrap.com/docs/4.3/getting-started/theming/#available-variables